packages/express/src/types.ts

```diff
@@ -327,7 +327,7 @@ export interface ExpressMiddlewareFactory<C> {
 /**
  * A symbol to store `MiddlewareContext` on the request object
  */
-export const MIDDLEWARE_CONTEXT = Symbol('loopback.middleware.context');
+export const MIDDLEWARE_CONTEXT = Symbol.for('loopback.middleware.context');
```

Comment:
Please add a comment mentioning that the symbol may be referenced by name before it is created. Using Symbol.for removes the module loading order sensitivity.

Reply:
Done!